tests/server/endpoints/test_orders_endpoint.py

```diff
@@ -67,11 +23,8 @@ def test_order_endpoint(
     # THEN the response should be successful
     assert response.status_code == HTTPStatus.OK
 
-    order_summary: OrderSummary = create_summaries(
-        orders=[order], analysis_summaries=[analysis_summary]
-    )[0]
-    # THEN the response should only contain the specified order
-    assert response.json == create_order_response(order=order, summary=order_summary).model_dump()
+    # THEN the response contains the specified order
+    assert response.json["id"] == order_id_to_fetch
```

Comment:
Is this not a strictly worse test now? Previously it was tested that the summary was exactly what we want. Now it only tests if it gives any summary for the correct order

Reply:
You are right, but I think it is worth it.

The structure of the summaries is bound to change and the cost of patching this test exceeds the value of verifying the structure of the summary. I think it is good enough to validate that A summary is generated for the correct order but I'll re-add it if you think differently.

Reply:
I still think it adds some value for now. But I do see your point. Do as you seem fit, I am fine either way.